projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.ts

```diff
@@ -43,14 +60,25 @@ export class CartItemComponent implements OnInit {
   parent: FormGroup;
 
   appliedProductPromotions$: Observable<PromotionResult[]>;
+  cartEntry$: Observable<OrderEntry>;
 
-  constructor(protected promotionService: PromotionService) {}
+  modalRef: ModalRef;
+  increment: boolean;
+
+  constructor(
+    protected promotionService: PromotionService,
+    protected cartService: CartService,
```

Comment:
That's a breaking change. Can you create multiple constructors and support cases when those new services won't be provided?